evap/evaluation/models.py

```diff
@@ -2017,3 +2017,10 @@ def send_textanswer_reminder_to_user(cls, user: UserProfile, evaluation_url_tupl
         body_params = {"user": user, "evaluation_url_tuples": evaluation_url_tuples}
         template = cls.objects.get(name=cls.TEXT_ANSWER_REVIEW_REMINDER)
         template.send_to_user(user, {}, body_params, use_cc=False)
+
+
+class VoteTimestamp(models.Model):
+    """A timestamp, when a user voted for a course."""
```

Comment:
Doesn't really match our [Terminology](https://github.com/e-valuation/EvaP/wiki/Terminology) and is a weird grammatically. Maybe just remove it?